nmap.cc

```diff
@@ -718,6 +720,11 @@ void parse_options(int argc, char **argv) {
       } else if (optcmp(long_options[option_index].name, "script-help") == 0) {
         o.scripthelp = true;
         o.chooseScripts(optarg);
+      } else if (optcmp(long_options[option_index].name, "script-timeout") == 0) {
+        l = tval2secs(optarg);
+        if ( l <= 0 )
+          fatal("Bogus value of Timeout given, timeout should be more than 0");
```

Comment:
Please mention the option name (`--script-timeout`) in the fatal message here. Just copy some language from another similar option.